AndroidBound/src/main/java/solutions/alterego/androidbound/android/interfaces/INeedsFontManager.java

```diff
@@ -0,0 +1,9 @@
+package solutions.alterego.androidbound.android.interfaces;
+
+
+public interface INeedsFontManager {
+
+    void setFontManager(IFontManager fontManager);
+
+    IFontManager getFontManager();
```

Comment:
I would move `getFontManager()` to the IViewBinder and just let `setFontManager()` stay in the `INeedsFontManager`, that way it can be used in subcomponents that need setting but not getting the manager. By default only `IViewBinder` would give you the font manager so as not to have multiples.